cloud/services/object_storage_buckets.go

```diff
@@ -21,7 +21,23 @@ func EnsureObjectStorageBucket(ctx context.Context, bScope *scope.ObjectStorageB
 		return nil, fmt.Errorf("failed to get bucket from region %s: %w", bScope.Bucket.Spec.Region, err)
 	}
 	if bucket != nil {
-		bScope.Logger.Info("Bucket exists")
+		bucketAccess, err := bScope.LinodeClient.GetObjectStorageBucketAccess(
```

Comment:
Nit: I think adding a log here to say we have an existing bucket and we are updating/updated bucket access would be beneficial